src/ui/window/BrowserWindow.qml

```diff
@@ -426,9 +426,7 @@ ApplicationWindow {
 
         MenuItem {
             text: "Fullscreen"
-            iconName: "navigation/fullscreen"
-            visible: !window.isFullScreen
-            height: visible ? undefined : 0
+            iconName: !window.isFullScreen ? "navigation/fullscreen" : "navigation/fullscreen_exit"
```

Comment:
@JohnPoison maybe `text: window.isFullScreen? "Show normal" : "Fullscreen"` too? What do you think?